can/parser.cc

```diff
@@ -292,6 +275,48 @@ void CANParser::UpdateValid(uint64_t sec) {
   }
 }
 
+double CANParser::get_value(uint32_t msg_addr, std::string &signal) {
+  auto state_it = message_states.find(msg_addr);
+  if (state_it == message_states.end()) {
+    fprintf(stderr, "CANParser: could not find message 0x%X in DBC\n", msg_addr);
+    assert(false);
+  }
+
+  MessageState state = state_it->second;
```

Comment:
this will make a deep copy. should be `const MessageState &state = state_it->second;`?